.github/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+services:
+  node:
+    labels:
+      app: spacetimedb
```

Comment:
what I don't love about this is that now we're not testing our "actual" `docker-compose.yml` or `Dockerfile`s. I don't think it's the biggest deal (we actually don't seem to test our root `Dockerfile` anyway), but I created https://github.com/clockworklabs/SpacetimeDBPrivate/issues/2275 to track that.

That said, that might be an argument for not creating these custom dockerfiles. I do appreciate having the smallest possible turnaround time for running tests, though, especially if we are moving this logic to `cargo ci` and expecting users to run it locally.